libvips/foreign/nsgifload.c

```diff
@@ -496,6 +454,8 @@ vips_foreign_load_nsgif_generate( VipsRegion *or,
 				p += 4;
 			}
 		}
+
+		prev_page = page;
```

Comment:
It'd be clearer if this was inside the `if(prev_page != page)` block above.

Reply:
Done!